rwengine/src/engine/GameWorld.cpp

```diff
@@ -461,7 +461,7 @@ GarageInfo* GameWorld::createGarage(const glm::vec3 coord0,
 
 Payphone* GameWorld::createPayphone(const glm::vec2 coord) {
     int id = payphones.size();
-    payphones.emplace_back(this, id, coord);
+    payphones.emplace_back(make_unique<Payphone>(this, id, coord));
```

Comment:
Didn't you forget include memory?

Reply:
It's in the header file

Reply:
I dont understand why it was merged before #476 even